services-geocoding/src/test/java/com/mapbox/api/geocoding/v5/MapboxGeocodingTest.java

```diff
@@ -45,6 +45,19 @@ public void sanity_batchGeocodeRequest() throws Exception {
     assertEquals(200, response.code());
   }
 
+  @Test
+  public void sanity_batchGeocodeSingleItemRequest() throws Exception {
+    MapboxGeocoding mapboxGeocoding = MapboxGeocoding.builder()
+            .mode(GeocodingCriteria.MODE_PLACES_PERMANENT)
```

Comment:
This test is a lot like the one below it with .mode(GeocodingCriteria.MODE_PLACES_PERMANENT) being the differentiator. 

If .mode(GeocodingCriteria.MODE_PLACES_PERMANENT) is left out there's an error message saying "Use getCall() for non-batch calls."  I wonder if it makes sense to change this message to also indicate that ".mode(GeocodingCriteria.MODE_PLACES_PERMANENT)" could be added as well in order to avoid the error condition.

Reply:
Makes sense, thank you! I've added clarification for the error message.